inst/app/global.R

```diff
@@ -1,5 +1,10 @@
 # Load packages
-require(saiUI)
+library(shiny)
+library(bslib)
+source('utils.R')
```

Comment:
Move `utils.R` to R directory inside `./inst/app/`. Files inside this structure is automatically sourced with shiny 1.5.0 and above